plugins/notifications/src/components/NotificationsSideBarItem/NotificationsSideBarItem.tsx

```diff
@@ -55,6 +111,40 @@ export const NotificationsSidebarItem = (props?: {
   const [refresh, setRefresh] = React.useState(false);
   const { setNotificationCount } = useTitleCounter();
 
+  const getSnackbarProperties = useCallback(
+    (notification: Notification) => {
+      const action = (snackBarId: SnackbarKey) => (
+        <>
+          <IconButton
+            onClick={() => {
+              if (notification.payload.link) {
+                window.open(notification.payload.link, '_blank');
```

Comment:
i am personally a _blank hater ;)

Anyway instead of onclick, it would make sense to see if you can delegate this to core-components Link whenever possible; https://github.com/backstage/backstage/blob/9dbf9f6cf4ae28352e25a069467d65316bf6762e/plugins/catalog/src/components/AboutCard/AboutCard.tsx#L234

then you get the chance of getting consistent click tracking behaviour and new-window policies etc. Maybe it could still have an onclick handler on it for those extra things it wants to happen, but not disabling the default action?